app/script/calling/entities/Flow.js

```diff
@@ -873,6 +879,7 @@ z.calling.entities.Flow = class Flow {
 
     const offer_options = {
       iceRestart: restart,
+      offerToReceiveVideo: this.offer_to_receive_video() ? true : undefined,
```

Comment:
I would add a comment there why it is important to use `undefined` instead of `false`.